MaxMind.MinFraud.UnitTest/Request/CreditCardTest.cs

```diff
@@ -34,26 +43,39 @@ public void TestIssuerIdNumberThatHasLetters()
         [Fact]
         public void TestLast4Digits()
         {
-            var cc = new CreditCard(last4Digits: "1234");
-            Assert.Equal("1234", cc.Last4Digits);
+            var cc2 = new CreditCard(last4Digits: "12");
+            Assert.Equal("12", cc2.Last4Digits);
```

Comment:
Maybe we should suppress the warning on the use of this, especially given how GitHub displays it.